### ISSUE TITLE
Caused by: com.mysql.cj.exceptions.CJException: Variable 'sql_mode' can't be set to the value of 'NO_AUTO_CREATE_USER'

### ISSUE THREAD

**Issue by iyappan-dcis** [apache/cloudstack#4549]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

Webapp

~~~

~~~

##### CLOUDSTACK VERSION

4.14

~~~

##### CONFIGURATION

MySQL Version - 8.0.22
MySQL Connector Java - 5.1.25



##### OS / ENVIRONMENT

Centos 7


##### SUMMARY
I get the below error while starting the cloudstack management service.

Caused by: com.mysql.cj.exceptions.CJException: Variable 'sql_mode' can't be set to the value of 'NO_AUTO_CREATE_USER'

**Comment by rhtyd** [comment]
@iyappan-dcis you want to wait until 4.15 to use MySQL8 with the latest mysql connector. There's a 4.15RC2 up for voting that you can test, see the dev ML https://markmail.org/message/jtsunp2z7rwl2zm7

**Comment by rhtyd** [comment]
sql_mode can be set via db.properties too at /etc/cloudstack/management

**Comment by iyappan-dcis** [comment]
@rhtyd Thanks for the update. How to we populate schema on the database, I don't see a command that populates the schema on database?

**Comment by uttruongson1995** [comment]
Are you fix this error ? @iyappan-dcis . i got same error

**Comment by weizhouapache** [comment]
this is fixed in 4.15/master.

mysql8 is not supported in 4.14

**Comment by rhtyd** [comment]
@uttruongson1995 @iyappan-dcis you're welcome to test the on-going 4.15RC3, see the thread - https://markmail.org/message/2hc34ete6se5tci6

**Comment by rhtyd** [comment]
This is fixed in 4.15.0.0, and master. Closing on this remark, pl upgrade/use the latest release.
